src/main/common/sensor_alignment.h

```diff
@@ -61,14 +61,22 @@ typedef union sensorAlignment_u {
     .yaw = DEGREES_TO_DECIDEGREES(YAW), \
 })
 
-#define CUSTOM_ALIGN_CW0_DEG         SENSOR_ALIGNMENT(  0,   0,   0)
-#define CUSTOM_ALIGN_CW90_DEG        SENSOR_ALIGNMENT(  0,   0,  90)
-#define CUSTOM_ALIGN_CW180_DEG       SENSOR_ALIGNMENT(  0,   0, 180)
-#define CUSTOM_ALIGN_CW270_DEG       SENSOR_ALIGNMENT(  0,   0, 270)
-#define CUSTOM_ALIGN_CW0_DEG_FLIP    SENSOR_ALIGNMENT(  0, 180,   0)
-#define CUSTOM_ALIGN_CW90_DEG_FLIP   SENSOR_ALIGNMENT(  0, 180,  90)
-#define CUSTOM_ALIGN_CW180_DEG_FLIP  SENSOR_ALIGNMENT(  0, 180, 180)
-#define CUSTOM_ALIGN_CW270_DEG_FLIP  SENSOR_ALIGNMENT(  0, 180, 270)
+#define CUSTOM_ALIGN_CW0_DEG         SENSOR_ALIGNMENT( 0,   0,   0 )
```

Comment:
I'd keep only 8 basic alignments and use macro for rest:
```
#define CUSTOM_ALIGN_CW(deg)   SENSOR_ALIGNMENT(0, 0, (deg));
#define CUSTOM_ALIGN_CW_FLIP(deg)   SENSOR_ALIGNMENT(0, 180, (deg));
#define CUSTOM_ALIGN_CW0_DEG CUSTOM_ALIGN_CW(0)
#define CUSTOM_ALIGN_CW90_DEG_FLIP CUSTOM_ALIGN_CW_FLIP(90)

Reply:
This would be 

```bash
// #define CUSTOM_ALIGN_CW0_DEG         SENSOR_ALIGNMENT( 0,   0,   0 )
#define CUSTOM_ALIGN_CW90_DEG        SENSOR_ALIGNMENT( 0,   0,  90 )
#define CUSTOM_ALIGN_CW180_DEG       SENSOR_ALIGNMENT( 0,   0, 180 )
#define CUSTOM_ALIGN_CW270_DEG       SENSOR_ALIGNMENT( 0,   0, 270 )
#define CUSTOM_ALIGN_CW0_DEG_FLIP    SENSOR_ALIGNMENT( 0, 180,   0 )
#define CUSTOM_ALIGN_CW45_DEG_FLIP   SENSOR_ALIGNMENT( 0, 180,  45 )
// #define CUSTOM_ALIGN_CW90_DEG_FLIP   SENSOR_ALIGNMENT( 0, 180,  90 )
#define CUSTOM_ALIGN_CW180_DEG_FLIP  SENSOR_ALIGNMENT( 0, 180, 180 )
#define CUSTOM_ALIGN_CW270_DEG_FLIP  SENSOR_ALIGNMENT( 0, 180, 270 )

#define CUSTOM_ALIGN_CW(deg)         SENSOR_ALIGNMENT( 0, 0, (deg) )
#define CUSTOM_ALIGN_CW_FLIP(deg)    SENSOR_ALIGNMENT( 0, 180, (deg) )
#define CUSTOM_ALIGN_CW0_DEG         CUSTOM_ALIGN_CW( 0 )
#define CUSTOM_ALIGN_CW90_DEG_FLIP   CUSTOM_ALIGN_CW_FLIP( 90 )
```

Reply:
It is IMO better to cascade it. That way, it is easier to change  CUSTOM_ALIGN internals
```
#define CUSTOM_ALIGN_CW90_DEG   CUSTOM_ALIGN_CW(90)
....